components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -2947,12 +3044,11 @@ private String handlePreConsent(OAuth2Parameters oauth2Params, AuthenticatedUser
                         + spTenantDomain + " for user: " + user.toFullQualifiedUsername());
             }
             if (LoggerUtils.isDiagnosticLogsEnabled()) {
-                Map<String, Object> configs = new HashMap<>();
-                configs.put("skipConsent", "true");
+                params.put("skipConsent", "true");
                 LoggerUtils.triggerDiagnosticLogEvent(OAuthConstants.LogConstants.OAUTH_INBOUND_SERVICE, params,
                         OAuthConstants.LogConstants.SUCCESS,
                         "'skipConsent' is enabled for the OAuth client. Hence consent claims not generated.",
-                        "generate-consent-claims", configs);
+                        "generate-consent-claims", null);
```

Comment:
is this change due to still using the deprecated method?

Reply:
Fixed with 358594f1d1df31773a93b3f1cf93ac0c8a1b119e